datasets/bkt/bkt.json

```diff
@@ -0,0 +1,367 @@
+{
+  "type": "dataset",
+  "id": "bkt",
+  "title": "bkt",
+  "status": "beschikbaar",
+  "description": null,
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "auth": "FP/MDW",
+  "tables": [
+    {
+      "id": "eigendomsrecht",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "bkBktEigendomsrecht"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "provenance": "bk_bkt_eigendomsrecht",
+            "type": "string"
+          },
+          "peildatumAfleiding": {
+            "type": "string",
+            "format": "date"
+          },
+          "statutaireNaam": {
+            "type": "string"
+          },
+          "indEigendomAmsterdam": {
+            "type": "boolean"
+          },
+          "kotToestandsdatum": {
+            "type": "string",
+            "format": "date"
+          },
+          "kotIdentificatie": {
```

Comment:
Kunnen we van deze indentificatie en volgnummer ook gewoon een 'normale' relatie maken? Zijn de overige velden 'gewoon' kadastraal objecten uit de brk?